doc/source/whatsnew/v0.21.0.txt

```diff
@@ -362,81 +362,54 @@ If installed, we now require:
    | Bottleneck   | 1.0.0           |          |
    +--------------+-----------------+----------+
 
-.. _whatsnew_0210.api_breaking.period_index_resampling:
 
-``PeriodIndex`` resampling
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-In previous versions of pandas, resampling a ``Series``/``DataFrame`` indexed by a ``PeriodIndex`` returned a ``DatetimeIndex`` in some cases (:issue:`12884`). Resampling to a multiplied frequency now returns a ``PeriodIndex`` (:issue:`15944`). As a minor enhancement, resampling a ``PeriodIndex`` can now handle ``NaT`` values (:issue:`13224`)
-
-Previous Behavior:
-
-.. code-block:: ipython
-
-   In [1]: pi = pd.period_range('2017-01', periods=12, freq='M')
-
-   In [2]: s = pd.Series(np.arange(12), index=pi)
-
-   In [3]: resampled = s.resample('2Q').mean()
+.. _whatsnew_0210.api_breaking.bottleneck:
 
-   In [4]: resampled
-   Out[4]:
-   2017-03-31     1.0
-   2017-09-30     5.5
-   2018-03-31    10.0
-   Freq: 2Q-DEC, dtype: float64
+Sum/Prod of all-NaN Series/DataFrames is now consistently NaN
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This commit is a purely reordering (putting eg sum of all NaN first, as this is included in the highlights).

The "list with missing values- indexing" might maybe also deserve a highlight?

Reply:
> The "list with missing values- indexing" might maybe also deserve a highlight?

Yeah, that seems reasonable.